### PR TITLE
issue #211 add: Fortran Lang governance 

### PR DESCRIPTION
**Pull request by Moksh45**

### CODE REVIEW

**Comment by henilp105** [comment]
#build_preview

File: source/community/governance.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+<p style="font-size:24px;color:#734f96;"><b>Administrators</b></p>
+
+The administrators are responsible for keeping Fortran-lang operational. They have access to the following assets
+
+- the Fortran-lang GitHub organization and can invite members, give write-access to projects, and create teams
+- the Fortran-lang Discourse and can give moderator privileges as well as moderate the forum
+- the fortran-lang.org domain
+- the [@fortranlang](https://twitter.com/fortranlang) Twitter account
+
+- the Fortran-lang [YouTube](https://www.youtube.com/user/Fortran%20Programming%20Language) channel
```
[the remark this answers is not in the document]
Comment (Moksh45) [reply source/community/governance.md]: I don't understand the meaning of "#build_preview." Could you please clarify or explain it in more detail?

**Comment by Moksh45** [comment]
I don't understand the meaning of "#build_preview." Could you please clarify or explain it in more detail?

**Comment by henilp105** [comment]
it is a command used to build a preview of the PR.

**Comment by Moksh45** [comment]
Okay, and what about my pull request? Is there any other change required?

**Comment by henilp105** [comment]
I have requested the above changes in my review for the PR, other than the above requested changes it looks good to me.

**Comment by Moksh45** [comment]
I am done with the changes you suggested. What about merging my pull request?

**Comment by henilp105** [comment]
There are some requested changes left in the above PR , ( marked as pending in this PR ) , other than the pending changes the PR looks good to me.

**Comment by Moksh45** [comment]
Did you only request to fix the link to the official YouTube Channel of Fortran Lang, which I have already fixed? Are there any other requested changes remaining?

**Comment by henilp105** [comment]
no, I have also requested the above marked changes in my review ( starting from `Can we please use Sphinx design for headings , It would help to translation of webpage.`) , I would like to request you to refer my review for the changes.  ( marked as pending in this PR review )

**Comment by Moksh45** [comment]
Okay, I made the required changes

**Comment by henilp105** [comment]
There are some requested changes left in the above PR , ( marked as pending in this PR ) . please refer this : 
image

**Comment by Moksh45** [comment]
I'm not sure why the "Can we please use Sphinx design for headings? It would help with webpage translation" issue doesn't appear in my problem section.
Screenshot from 2023-07-09 01-04-28

**Comment by Moksh45** [comment]
I believe I have incorporated the Sphinx design for headings. Could you please review my pull request to confirm if I have implemented it correctly?

**Review by henilp105 — COMMENTED** [review]
Other than above requested changes, looks good to me.

**Comment by Moksh45** [comment]
I have implemented all the necessary modifications, kindly review them.

File: source/community/governance.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+Contributors
+:::
+
+Everybody is welcome to contribute to Fortran-lang and its projects. Providing and discussing ideas as well as developing concepts and designs are integral contributions to any project. All contributors can steer and add to the long-term vision of the Fortran-lang projects.
+
+:::{div} sd-text-left sd-fs-2 sd-font-weight-bold sd-text-primary
+MIT License
+:::
+
+To the extent possible under law, the authors have waived all copyright and related or neighboring rights to the Fortran-lang community governance document, as per the CC-0 public domain dedication.
```
Comment (henilp105) [inline source/community/governance.md:71]: Can we change the license to MIT as Most of the Fortran - Lang projects are under MIT License.

Comment (Moksh45) [reply source/community/governance.md:71]: I'm sorry, but I'm having trouble understanding your specific requirements. Could you please provide further clarification what you're looking for?

Comment (henilp105) [reply source/community/governance.md:71]: we cannot use CC-0 public domain license , Most of the Fortran - Lang projects are under MIT license.

Comment (Moksh45) [reply source/community/governance.md]: I believe it is now correct

Comment (milancurcic) [reply source/community/governance.md]: I think either is fine. MIT is more common for software whereas CC-* licenses are more common for docs.

Comment (henilp105) [reply source/community/governance.md]: > the authors have waived all copyright and related or neighboring rights to
as Most of the Fortran - Lang projects are under MIT license , this is an incorrect statement.

**Comment by henilp105** [comment]
#build_preview

File: source/community/governance.md (hunk 2 above)
Comment (Moksh45) [reply source/community/governance.md]: Could someone kindly advise me on what actions I should take?  I have no knowledge about licenses 😅

Comment (henilp105) [reply source/community/governance.md]: it seems to be good for the docs.

**Review by henilp105 — APPROVED** [review]
Thanks @Moksh45 , Looks good to me.

**Comment by Moksh45** [comment]
I'm actually searching for good first issues. If there are any available, please let me know.

**Pull request merged**